### PR TITLE
Issue 433 - Catalog endpoint does not return usage info

### PR DESCRIPTION
**Pull request by javier-gomez-hs**

### CODE REVIEW

File: src/main/java/org/killbill/billing/client/model/TieredBlock.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+public class TieredBlock {
+
+    private final String unit;
+    private final String size;
+    private final String max;
+    private final List<Price> prices;
+
+    @JsonCreator
+    public TieredBlock(@JsonProperty("billingPeriod") final String unit,
```
Comment (pierre) [inline src/main/java/org/killbill/billing/client/model/TieredBlock.java]: Typo: `"unit"`

File: src/main/java/org/killbill/billing/client/model/Tier.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Tier {
+
+    private final List<TieredBlock> blocks;
+
+    @JsonCreator
+    public Tier(@JsonProperty("tiers") final List<TieredBlock> blocks) {
```
Comment (pierre) [inline src/main/java/org/killbill/billing/client/model/Tier.java]: Aren't we also missing limits and prices for the [capacity in arrear](https://github.com/killbill/killbill/blob/master/catalog/src/test/resources/UsageExperimental.xml#L170-L193) use-case?

Comment (javier-gomez-hs) [inline src/main/java/org/killbill/billing/client/model/Tier.java]: Done! Good catch!

File: src/main/java/org/killbill/billing/client/model/Unit.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Unit {
+
+    private final String name;
+
+    @JsonCreator
+    public Unit(@JsonProperty("name") final String name) {
```
Comment (pierre) [inline src/main/java/org/killbill/billing/client/model/Unit.java]: Same question as the PR in Kill Bill: do we need the `Unit` object?

Comment (javier-gomez-hs) [inline src/main/java/org/killbill/billing/client/model/Unit.java]: @pierre 
We probably don't need the Unit object!
I've removed it.

Thanks!

**Pull request merged**